Cubical/Algebra/NAMING.md

```diff
@@ -2,6 +2,11 @@ NAMING
 ======
 
 This file provides a guide for naming definitions in the Algebra folder.
+You may divert from the naming scheme for sufficiently 'local' names,
+e.g. names which are ```private``` or in a ```where``` clause.
```

Comment:
```suggestion
e.g. names which are ```private``` or in a ```where```/```let``` clause.
```

Reply:
With the cool "suggestion" feature you can just click "Commit suggestion" btw and not change anything locally